### ISSUE TITLE
Tick Values

### ISSUE THREAD

**Issue by marino79** [olifolkerd/tabulator#1431]
Hello and sorry for my poor english,
Thanks a lot for tabulator, a really powerful tools.
I have a little problem using the beta 4.0 version; I'd like to change the tick's behavior,  setting a difference between 0 value and NULL value. it's possible?!?
it would be very useful for me to visually distinguish a negative value from one that I still have to insert, for example!
Thanks in advance for your answer !

**Comment by olifolkerd** [comment]
Hey @marino79 

Are you talking about the way a tick is displayed by a formatter or the value after editing with an editor?

Cheers

Oli :0

**Comment by marino79** [comment]
Hi Oli,
I’m talking about the way a tick is displayed by default formatter: the green tick for 1 o true values and the red tick for other value.i would like see the red tick for 0 value(or false) and nothing for values empty or null

**Comment by olifolkerd** [comment]
Hey @marino79 

You would need to use a custom formatter for this.

Checkout the [Formatter Documentation](http://tabulator.info/docs/3.5#formatting) for more info.

Cheers

Oli :)

**Comment by marino79** [comment]
Ok but i have to inplement a new one formatter or i can improve the tickCross?

**Comment by olifolkerd** [comment]
Hey @marino79 

I liked your idea so much i have added two new options to the ***tickCross*** formatter.

The ***allowEmpty*** param will make the tick cross formatter show an empty cell instead of a cross if the cell has a value of ***undefined*** ***null*** or ***""***

The ***allowTruthy*** param will make the tick cross formatter show a tick if the cell has any value that can be considered truthy

``js
{title:"Likes Cheese", field:"cheese", formatter:"tickCross", formatterParams:{allowEmpty:true, allowTruthy:true}}
```

These are available on the 4.0 branch and will be available in tommorow update.

Thanks for the feedback.

Cheers

Oli :)
